src/libgeodecomp/loadbalancer/loadbalancer.h

```diff
@@ -4,14 +4,20 @@
 #include <libgeodecomp/misc/stdcontaineroverloads.h>
 #include <vector>
 
+#if defined(LIBGEODECOMP_WITH_HPX)
+#include<hpx/config.hpp>
+#elif !defined(HPX_COMPONENT_EXPORT)
+#define HPX_COMPONENT_EXPORT
```

Comment:
Well, actually, this macro is needed always, even if HPX is not used. I was too lazy to introduce something equivalent for LGD, so I reused the HPX export macros...